backend/app/literature/tests/test_002_resource.py

```diff
@@ -90,6 +90,74 @@ def test_update_Resource():
     assert res['abstract'] == '3'
 
 
+def test_resource_create_large():
+    xml = {
+        "abbreviation_synonyms": ["Jackson, Mathews, Wickens, 1996"],
+        "cross_references": [
+            {
+                "curie": "FB:FBrf09999999",
```

Comment:
8-digit FBrf ID is atypical - intentional?

Reply:
Changed,

Reply:
It did not matter, there is no qc on this, but yep its better to keep the test data close to what is expected. thanks.